src/battle-scene.ts

```diff
@@ -2651,7 +2650,7 @@ export default class BattleScene extends SceneBase {
       return encounter;
     }
 
-    // Common / Uncommon / Rare / Super Rare
+    // Common / Great / Ultra / Rogue
     const tierWeights = [64, 40, 21, 3];
```

Comment:
Couldn't you define those numbers on the `MysteryEncounterTier` enum too? Or would that cuase side effects?

```ts
export enum MysteryEncounterTier {
  COMMON=64,
  GREAT=40,
  ULTRA=21,
  ROGUE=3,
  MASTER=0, // Not currently used
}
```

Reply:
[These numbers are decremented over time throughout a session](https://github.com/AsdarDevelops/PokeRogue-Events/pull/97/files/670013a1076562698db6220777577d65772d0c4f#diff-4b1a5f9160d1d8aecce0db85774f20e3c254f97c27cc0d0473911d5cbd565396R2657-R2664), so wouldn't really work well I think.

Reply:
not a bad idea